src/commands/deploy.ts

```diff
@@ -172,7 +176,9 @@ export default class Deploy extends DeployCommand {
     const components = args.configs_or_components;
 
     const interfaces_map = DeployUtils.getInterfacesMap(flags.interface);
-    const component_secrets = DeployUtils.getComponentSecrets(flags.parameter, flags.secrets);
+    const component_secrets = DeployUtils.getComponentSecrets(flags.secret, flags['secret-file']);
+    const component_parameters = DeployUtils.getComponentSecrets(flags.parameter, flags['secret-file']); // TODO: 404: remove
```

Comment:
Does this still need to be removed?

Reply:
Yes, but since these things are just deprecated for now, I marked them to be resolved in the future with this ticket

https://gitlab.com/architect-io/architect-cli/-/issues/404